chcemvediet/apps/anonymization/content_types.py

```diff
@@ -0,0 +1 @@
+PDF_CONTENT_TYPE = "application/pdf"
```

Comment:
Zvykneme vsetky stringy pisat ako unicode `u'...'`, lebo ked niektore stringy su unicode a niektore nie, tak obcas byva s nimi problem.

Tiez zvykneme pouzivat `'` nie `"`". Na tom sice nezalezi, ale nech pouzivame vsade rovnaky zapis. `"` pouzijeme iba ak `'` z nejakeho dovodu nepomozeme pouzit (napriklad string obsahuje vo vnutri `'`).

Pls oprav to aj na ostatnych miestach v PR.